### ISSUE TITLE
Remove the hardcoded method count for t("")

### ISSUE THREAD

**Issue by RadicalMonkey** [orhanobut/logger#27]
For the t("") function it should use the the user defined value and not the default value of 2

**Comment by orhanobut** [comment]
t("") means settings tag, not the method count. As default method count is 2 unless it is set explicitly. 
<pre>
t(0) or t(TAG,0)
</pre>

**Comment by RadicalMonkey** [comment]
Yes I understand. I've set the method count in my Application class to 0 but ```t()``` does not respect that.

**Comment by orhanobut** [comment]
can you show me your implementation please?

**Comment by orhanobut** [comment]
Hmm I get the issue, yes you are right, that should be a check there.

**Comment by RadicalMonkey** [comment]
In my Application class I have:
```java
private void configureLogger(boolean isDebug) {
        Logger.init("Khimera")
                .setMethodCount(1)
                .hideThreadInfo()
                .setLogLevel(isDebug ? LogLevel.FULL : LogLevel.NONE);
    }
```

Where I want to the logger with the extra tag I have:
```java
Logger.t("App").i(error.getMessage());
```

**Comment by orhanobut** [comment]
I'll fix this issue, indeed this is a bug.

**Comment by RadicalMonkey** [comment]
Thanks, great library by the way :)

**Comment by orhanobut** [comment]
Thanks! This issue should be fixed with v1.9, I just released a version for that.

**Comment by RadicalMonkey** [comment]
Works great, thanks!
